code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -445,7 +435,7 @@
 /obj/item/tool/weldingtool/simple
 	name = "\improper ME3 hand welder"
 	desc = "A compact, handheld welding torch used by the marines of the United States Colonial Marine Corps for cutting and welding jobs on the field. Due to the small size and slow strength, its function is limited compared to a full-sized technician's blowtorch."
-	max_fuel = 5
+	max_fuel = 10
```

Comment:
Undocumented change that probably doesn't need to change.

Reply:
will add to the changelog . this is needed so non engi classes can heal themselves when they get prostetic limbs at least for a limited amount (5 uses ) . sincerely i dont think this affects balance at all , nobody touches that torch